tests/test_chatbot.py

```diff
@@ -117,6 +122,21 @@ def validate(self, completion):
         return completion
 
 
+@pytest.fixture(scope="session")
+def database_file(tmp_path_factory):
+    # Create a temporary directory and file for the database
+    db_file = tmp_path_factory.mktemp("data").joinpath("documents.db")
+
+    # Generate the actual embeddings
+    documents_manager = DocumentsDB(db_file)
+    documents = pd.read_csv(DOCUMENTS_CSV)
+    documents = generate_embeddings(documents, documents_manager)
```

Comment:
do we really want to regenerate the embeddings every time ?

Reply:
We are only generating 10 embeddings, so very little cost compared to every GPT call we do anyway, and it makes it such that the embeddings are always "fresh"